assets/src/components/editing/editor/normalizers/normalizer.ts

```diff
@@ -38,13 +38,12 @@ export function installNormalizer(editor: Editor, context: NormalizerContext = {
           Transforms.wrapNodes(editor, Model.p(), { at: path });
           return;
         }
```

Comment:
These normalizers were not running on top level elements. Pretty sure they weren't meant to be in the else block.

I had this change in it's own PR and had asked for comments in case I was missing something big there:
https://github.com/Simon-Initiative/oli-torus/pull/2890